docs/apps/activitystream.md

```diff
@@ -0,0 +1,122 @@
+# activitystream
+
+The django-ansible-base `activitystream` app provides an audit log of changes
+that happen to models within the system.
+
+## Consumer Docs
+
+To make use of `activitystream` in your application, first add
+`ansible_base.activitystream` to your `INSTALLED_APPS`.
+
+Now in your models you can:
+
+```python
+from ansible_base.activitystream.models import AuditableModel
+```
+
+... and make your model inherit from `AuditableModel`. In theory, this is all
+that is required. When your application starts up, `activitystream` will ask
+Django to list all the models it knows about. It will then filter this list for
+ones that inherit from `AuditableModel` and call the
+`AuditableModel::connect_signals` class method which registers the appropriate
+signals.
+
+You can exclude fields from being included in the changes logged by
+`activitystream` by setting the `activity_stream_excluded_field_names` class
+variable in your model.
+
+### What changes look like
```

Comment:
What does it look like when a ForeignKey is changed? Also, a extremely minimal example of this JSON would be nice.

Reply:
https://github.com/ansible/django-ansible-base/issues/250